src/test/ui/suggestions/core-std-import-order-issue-83564.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0433]: failed to resolve: use of undeclared type `NonZeroU32`
+  --> $DIR/core-std-import-order-issue-83564.rs:5:14
+   |
+LL |     let _x = NonZeroU32::new(5).unwrap();
+   |              ^^^^^^^^^^ not found in this scope
+   |
+help: consider importing this struct
+   |
+LL | use std::num::NonZeroU32;
+   |
+
+error: aborting due to previous error
+
+For more information about this error, try `rustc --explain E0433`.
```

Comment:
Hmm, actually, this is the same error I get on stable, so this may not be testing what it should.

Reply:
For some reason, Rust 2018 or higher is required to reproduce the original bug, so I added `// edition:2018`.